src/app/components/search/SearchResults.js

```diff
@@ -317,6 +318,19 @@ class SearchResultsComponent extends React.PureComponent {
 
     const isProject = !!this.props.project;
 
+    const smoochBotInstalled = (
```

Comment:
is this condition present elsewhere in the code? I suggest to make a helper `isBotInstalled(team, identifier)` out of that.

Reply:
done